src/inspection/README.md

```diff
@@ -0,0 +1,27 @@
+# Inspection package
+
+This package is responsible for running two scripts used for testing the car. One of them performs the autonomous mission described in the rules: "T14.11.2 The inspection mission is defined by slowly spinning the drivetrain and actuating the steering system with a sine wave while the vehicle is jacked up and all wheels are removed. After 25 s to 30 s the AS must transition to 'AS Finished'". The other script is used to accelerate the car to a maximum speed and then braking.
```

Comment:
The other script is another mission in the rules called EBS test. You should probably also mention that the scripts can be parameterized via file